tests/test_fixtures/rule_based_profiler/quentin_user_workflow_verbose_profiler_config.yml

```diff
@@ -42,15 +43,17 @@ rules:
         metric_domain_kwargs: $domain.domain_kwargs
         metric_value_kwargs:
           quantiles: $variables.quantiles
-          allow_relative_error: linear
+          allow_relative_error: $variables.allow_relative_error
```

Comment:
Hi @alexsherstinsky one question about this value : where did this change come from?

Reply:
From our synchronous discussion:  previous version specified configuration incorrectly — detected upon studying `https://greatexpectations.io/expectations/expect_column_quantile_values_to_be_between` closely

Reply:
@Shinnnyshinshin In addition: I put that value into the `variables` section so that it could be changed, if needed, more easily.  Thanks!